File: java/sprint1_nonfinals/H.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;


public class H {
    private static String sumOfBinaries(String a, String b) {
        // Ваше решение
    }
    public static void main(String[] args) {
        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
            String a = reader.readLine();
            String b = reader.readLine();
            System.out.println(sumOfBinaries(a, b));
        }
    }
}

File: csharp/sprint1_nonfinals/H.cs
using System;
using System.IO;

public class H
{
    private static TextReader reader;
    private static TextWriter writer;

    private static string SumOfBinaries(string a, string b)
    {
        // Ваше решение
    }

    public static void Main(string[] args)
    {
        reader = new StreamReader(Console.OpenStandardInput());
        writer = new StreamWriter(Console.OpenStandardOutput());

        var a = reader.ReadLine();
        var b = reader.ReadLine();
        writer.WriteLine(SumOfBinaries(a, b));

        reader.Close();
        writer.Close();
    }
}
